pdc_client/plugins/auth_group_resource_permissions.py

```diff
@@ -0,0 +1,101 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2015 Red Hat
+# Licensed under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+#
+
+from __future__ import print_function
+
+import json
+
+from pdc_client.plugin_helpers import (PDCClientPlugin,
+                                       extract_arguments,
+                                       add_parser_arguments,
+                                       add_create_update_args)
+
+
+class AuthGroupResourcePermissionPlugin(PDCClientPlugin):
+    def register(self):
+        self.set_command('auth/group-resource-permissions')
+        list_parser = self.add_action('list', help='list all groups and their resource permission')
+        add_parser_arguments(list_parser, {'resource': {},
+                                           'permission': {},
+                                           'group': {}},
+                             group='Filtering')
+        list_parser.set_defaults(func=self.list_group_resource_permission)
+
+        info_parser = self.add_action('info', help='display details about a group resource permission')
+        info_parser.add_argument('id', metavar='ID')
+        info_parser.set_defaults(func=self.group_resource_permission_info)
+
+        update_parser = self.add_action('update', help='update an existing group resource permission')
+        update_parser.add_argument('id', metavar='ID')
+        self.add_create_update_argument(update_parser, required=True)
+        update_parser.set_defaults(func=self.group_resource_permission_update)
+
+        create_parser = self.add_action('create', help='create new group resource permission')
+        self.add_create_update_argument(create_parser, required=True)
+        create_parser.set_defaults(func=self.group_resource_permission_create)
+
+        delete_parser = self.add_action('delete', help='delete an existing group resource permission')
+        delete_parser.add_argument('id', metavar='ID')
+        delete_parser.set_defaults(func=self.group_resource_permission_delete)
+
+    def add_create_update_argument(self, parser, required=True):
+        required_args = {'resource': {}, 'permission': {}, 'group': {}}
+        add_create_update_args(parser, required_args, {}, required)
+
+    def print_group_resource_permissions(self, args):
+        fmt = '{0:<5} {1:20} {2:90} {3}'
+        start_line = True
+        for obj in args:
+            if start_line:
+                start_line = False
+                print(fmt.format('ID', 'Group', 'Resource', 'Permission'))
+                print()
+            print(fmt.format(obj['id'], obj['group'], obj['resource'], obj['permission']))
+
+    def list_group_resource_permission(self, args):
+        filters = extract_arguments(args)
+        results = self.client.get_paged(self.client['auth/group-resource-permissions']._, **filters)
+        if args.json:
+            print(json.dumps(list(results)))
+            return
+        if results:
```

Comment:
Why judge results here but not in above? What is the difference in judging results between json and not json?